### ISSUE TITLE
Time format in loot history tooltip has weird format

### ISSUE THREAD

**Issue by Numielle** [evil-morfar/RCLootCouncil_Classic#11]
**Expected behaviour:**
I see the last few items assigned to a player sorted chronologically. The time of receival is formatted: _x months y days ago_

**Actual behaviour:**
I see the last few items assigned to a player sorted chronologically. The time of receival is formatted: _y days x months ago_:
![WoWScrnShot_022820_194551](https://user-images.githubusercontent.com/5376958/75578079-7b975980-5a63-11ea-81bc-1e9029bbd7a6.jpg)

**Comment by Jorvanus** [comment]
I agree, it's a bit strange and def more difficult to read than needed.   It's still like this now also.  +1 for a change here.

**Comment by evil-morfar** [comment]
Changed in v0.24.0
